backfiller/backfiller/main.py

```diff
@@ -210,6 +244,8 @@ def run(self):
 					failures += 1
 				delay = common.jitter(TIMEOUT * 2**failures)
 				self.logger.exception('Getting nodes failed. Retrying in {:.0f} s'.format(delay))
+				host = [s.split('=')[-1] for s in self.connection.dsn.split() if 'host' in s][0]
```

Comment:
i'm worried about this line failing, especially since it only runs on error. maybe wrap in a try and set host='<unknown>' on error?